lib/main_common.pm

```diff
@@ -917,7 +917,12 @@ sub load_consoletests {
     loadtest "console/textinfo";
     loadtest "console/rmt" if is_rmt;
     loadtest "console/hostname" unless is_bridged_networking;
-    replace_opensuse_repos_tests if get_var('DISABLE_ONLINE_REPOS');
+    # Run zypper info before as tests source repo
+    loadtest "console/zypper_info";
```

Comment:
In any case we have to agree that this is not correct in the long-run as by having it here we are relying on content of the published repo so we are not testing the clean snapshot along with all prbomles that we want to fix with the work on the ticket you are creating this PR for and you do not even mention that in the comment in L920. Maybe syncing a single package is easier than we think because in the end what we need to control is just rsync. E.g. see https://stackoverflow.com/a/9952100/5031322 about rsync options.